tango_simlib/utilities/helper_module.py

```diff
@@ -119,7 +123,7 @@ def append_device_to_db_file(
     db = Database(db_file_name)
     # Patched the property dict to avoid a PyTango bug
     patched = dict(
-        (key, value if value != "" else " ") for key, value in properties.items()
+        (key, value if value != "" else " ") for key, value in list(properties.items())
```

Comment:
This can be rewritten as suggested. 


## TL;DR
>With CPython 2.7, using dict() to create dictionaries takes up to
6 times longer and involves more memory allocation operations than the
literal syntax. Use {} to create dictionaries, especially if you
are pre-populating them, unless the literal syntax does not work for
your case.

Ref: [The Performance Impact of Using dict() Instead of {} in CPython 2.7](https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/)

In Python you should always use literal syntax whenever possible, this improves performance and efficiency - See image below.

![image](https://user-images.githubusercontent.com/7910856/70213660-8aab8b80-1742-11ea-8167-b4e6c97ad820.png)

```suggestion
    patched = {
        key: value if value != "" else " " for key, value in properties.items()
    }
```